.github/workflows/test.yaml

```diff
@@ -25,7 +25,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.9", "3.10", "3.11", "3.12"]
-        environment-file: [ci/environment.yml, ci/environment_released.yml]
```

Comment:
Is this intended?

Reply:
Yes

We are currently incompatible with released dask dask

Reply:
Makes sense, in an ideal world we should have split that into a separate PR.

Reply:
I actually merged this already in another PR before this one, not sure why the diff is still the way it is